org.osgi.security.sysatt110.test/src/org/osgi/security/sysatt110/test/SysAtt110Test.java

```diff
@@ -1,54 +1,47 @@
 package org.osgi.security.sysatt110.test;
 
-import org.osgi.framework.BundleContext;
+import static org.osgi.framework.FrameworkUtil.getBundle;
+
+import java.io.File;
+
+import org.junit.After;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
-import org.osgi.framework.BundleActivator;
-
+import org.osgi.framework.BundleContext;
 import org.osgi.security.util.api.Util;
 import org.osgi.util.tracker.ServiceTracker;
 
-import java.io.File;
-
 /**
  * 
  **/
 
-public class Activator implements BundleActivator
+public class SysAtt110Test
 {
-	private static BundleContext bundleContext;
-	private ServiceTracker serviceRef;
+	private BundleContext bundleContext = getBundle(SysAtt110Test.class).getBundleContext();
```

Comment:
But then you would need to add that Junit runner to every project so that Eclipse can find it, no? That doesn't seem helpful either.

But in any case, for these test case, now, we should use the setBundleContext methods support to receive the BundleContext.